rescript/plugin_setup.py

```diff
@@ -634,6 +635,31 @@
 )
 
 
+plugin.methods.register_function(
+    function=get_ncbi_data,
+    inputs={},
+    parameters={
+        'query': Str,
+        'accession_ids': Metadata},
+    outputs=[('sequences', FeatureData[Sequence]),
+             ('taxonomy', FeatureData[Taxonomy])],
+    input_descriptions={},
+    parameter_descriptions={
+        'query': 'Query on the NCBI Nucleotide database',
```

Comment:
Perhaps a short example here? Like: "txid1239[ORGN] NOT environmental[Title]", or whatever your favorite search is. Or perhaps add a links to these:

- https://www.ncbi.nlm.nih.gov/books/NBK49540/
- https://www.ncbi.nlm.nih.gov/books/NBK3837/

There maybe better ones...

Reply:
I like both of these ideas... and you could add this to the `description` instead of here if you decide to add an extended description.

Reply:
alternatively, maybe we should create a little tutorial/cookbook in the README.md file and just direct users to the repo for more details (a cookbook would be really useful for common queries like the 16S refseqs, and would help me test this PR too)

Reply:
Why don't we add it to the existing [tutorial](https://forum.qiime2.org/t/processing-filtering-and-evaluating-reference-sequence-data-with-rescript/15494)?

Reply:
sure, adding existing examples to the existing tutorial works for now. Would be nice to get a dedicated tutorial/split up that monstrously long tutorial into thematic subjects... but that is not a prerequisite to merging this PR!

Reply:
Good call. RESCRIPt is going to do a bunch of different things for different sets of users.